metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -3168,11 +3214,63 @@ def emit_site_container(self):
             sub_types=[c.SITE],
         )
 
+    def _fetch_groups(self):
+        for group in TSC.Pager(self.server.groups):
+            self.group_map[group.id] = group
+
+    def _filter_capabilities(self, capabilities):
+        if not self.config.group_ingestion:
+            return ""
+
+        allowed_capabilities = [
+            key for key, value in capabilities.items() if value == "Allow"
+        ]
+        if self.config.group_ingestion.displayed_capabilities:
+            allowed_capabilities = [
+                value
+                for value in allowed_capabilities
+                if value in self.config.group_ingestion.displayed_capabilities
+            ]
+        return ", ".join(allowed_capabilities)
+
+    def _create_workbook_properties(
+        self, permissions: List[PermissionsRule]
+    ) -> Optional[Dict[str, str]]:
+        if not self.config.group_ingestion:
+            return None
+
+        groups = []
+        for rule in permissions:
+            if rule.grantee.tag_name == "group":
```

Comment:
are groups the only possibilities? or can individual users also get grants - and that's something that's just not supported by this PR?

Reply:
In theory, there could be individual users that have access to the Tableau assets. However, from a request access perspective only groups really make sense to display in the central catalog, that's why it's currently not supported by this PR.